modules/monitoring-setting-query-log-file-for-prometheus.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="setting-query-log-file-for-prometheus_{context}"]
+:_content-type: PROCEDURE
+= Enabling the query log file for Prometheus
+
+[role="_abstract"]
+You can configure Prometheus to log all queries run by the engine to a log file.
```

Comment:
Is `queries run` a known terminology/phrase? I'm reading it like the queries have been ran. 

Would a rephrase like this keep the meaning but make reading the sentence easier? `You can configure Prometheus to log all the queries that have been run by the engine.` 

This is wordier, I know, but let me know if you think it helps readability. I could be way overthinking it and the original sentence might read perfectly fine as is, in which case, I trust your judgement :) 

Note, do we need to say `to a log file` at the end if we already said that Prometheus logs it?

Reply:
- Per your suggestion, I'm revising to say: `You can configure Prometheus to write all queries that have been run by the engine to a log file.`
- Re your last comment, we need to say "to a log file" because logging information could in theory be output (e.g. via the `tail` command) directly to the screen rather than to a file.